frontend/src/app/shared/components/page-not-found/page-not-found.component.html

```diff
@@ -1,24 +1,23 @@
 <main>
-  <!-- LARGE AND MEDIUM SCREENS -->
-  <div class="sm-show">
+  
     <div class="p2"></div>
     <div class="flex">
       <span class="flex-grow"></span>
       <mat-card class="col col-6 center">
         <div class="p4"></div>
-        <mat-card-title> PAGE NOT FOUND </mat-card-title>
-        <mat-card-subtitle> The requested resource was not found </mat-card-subtitle>
+        <mat-card-title> 404 </mat-card-title>
+        <div class="separator">
+        <mat-card-subtitle> Sorry - the resource you have requested does not exist </mat-card-subtitle>
+        <p>You may have entered an invalid URL. If you believe this to be an error, please 
+          <a href="https://docs.google.com/forms/d/e/1FAIpQLSdScFpcZi7G5sjPo6RKsg6BYbV6PFiH9nMWoawRYIbAyRucnA/viewform">
```

Comment:
@TheBroMoe Could you transfer ownership of this form to jared.rewerts@edmonton.ca?

Reply:
You mean like instead of redirecting to the form it redirects to your email? because if yes I can definitely do that!

Reply:
I can't actually access that form. I just shared a new form with you. Would you mind updating it to use that one?

Reply:
New form has been added! I also modified it a bit, is there anything specific you also want people to talk about in the form?

Reply:
I'll probably touch up the form a bit, but could you use https://docs.google.com/forms/d/e/1FAIpQLScxdY0tJJXKGAPvuCtwL23TVc72G14TCNMxsWdWdZAn57dh2g/viewform? Right now, it tries to open the edit view.

Reply:
I get "resource unavailable" when I try to open the link. Is there a new link to use instead?

Reply:
Hmm... Let me update the permissions. It should work now.

Reply:
form is now updated